Styles/styles.css

```diff
@@ -0,0 +1,111 @@
+h1 {
+    position: absolute;
+    width: 306px;
+    height: 47px;
+    left: 600px;
+    top: 120px;
+}
+hr {
+    position: absolute;
+    width: 100px;
+    height: 4px;
+    left: 670px;
+    top: 187px;
+    background-color: #F5BD41;
+}
+div {
+    display: inline-block;
+}
+.container {
+    position: absolute;
+    width: 296px;
+    height: 183px;
+}
+
+.support {
+    position: absolute;
+    width: 100px;
+    height: 100px;
+    left: 202px;
+    top: 271px;
+}
+
+.communication {
+    position: absolute;
+    width: 100px;
+    height: 100px;
+    left: 513px;
+    top: 271px;
+}
+.booking {
+    position: absolute;
+    width: 100px;
+    height: 100px;
+    left: 826px;
+    top: 271px;
+}
+.reviews {
+    position: absolute;
+    width: 100px;
+    height: 100px;
+    left: 1138px;
+    top: 271px;
+}
+.text1 {
+    position: absolute;
+    width: 141px;
+    height: 28px;
+    left: 181px;
+    top: 419px;
+
+    font-family: 'Roboto';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
```

Comment:
There are the same font styles for the text1, text2, text3, text4 in the rows 61-66. You can add some class with font styles and use it in every element you need.
Also you have no need to use position absolute here.